pyqms/adaptors.py

```diff
@@ -152,7 +153,7 @@ def _parse_evidence_and_format_fixed_labels(data=None):
             converted_value = PARAM_TYPE_LOOKUP[k](v)
             r["params"][k] = converted_value
 
-    cc_factory = pyqms.chemical_composition.ChemicalComposition()
+    cc_factory = pyqms.ChemicalComposition()
```

Comment:
the chemical Composition should not be imported in the pyqms name space, so should read something like chemical_composition.ChemicalComposition() or so